test/unit_test/character_sheet_unit_tests/RaceUnitTests.java

```diff
@@ -0,0 +1,27 @@
+package unit_test.character_sheet_unit_tests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import rpg_database.character_sheet.Background;
+import rpg_database.character_sheet.CharacterSheet;
+import rpg_database.character_sheet.Fields;
+import rpg_database.character_sheet.Race;
+
+public class RaceUnitTests {
+
+	@Test
+	public void testGetRaceName() {
+		assertEquals("Dwarf", Race.DWARF.toString());
+	}
+
+	@Test
+	public void testSetBackgroundGetRace() {

```

Comment:
Ez maradhat így, de:
1. ez nem ide való. Ez a Background setSelfInSheet-jének a funkcionalitása, tedd a Background tesztjei közé.
2. szerintem biztosamibiztos alapon az sem árt ha explicite beállítod a HUMAN-t a RACE fieldjében a karakterlapnak. Ha bármi változna, akkor ez vagy elfailel vagy beállítja humannek ha teheti. Mivel a RACE kívülről állítható lesz a Background-Race trivialitás hiánya miatt ezért gondolom nem tiltod majd le a kívülről állíthatóságot.
3. a 2-es pontból kiindulva viszont ha kívülről állítom a RACE-t akkor nincs rá semmilyen csekkolás hogy azt megtehetem -e vagy sem. Ezt tesztelni kellene a tiltás helyett majd, a karakterlap tesztjeiben, vagy itt a Race-nél. Attól függ hogy a CharacterSheet vagy a Race fogja implementálni a Background által engedett racekhez való checkolási mechanikát. Szerintem az utóbbi szerencsésebb, de rád bízom

Reply:
ok